tardis/apps/publication_forms/static/publication-form/publication_form.js

```diff
@@ -374,6 +374,8 @@ app.controller('publicationFormCtrl', function ($scope, $log, $http, ngDialog, $
             $scope.infoMessage = "";
             $scope.currentPageIdx--;
             $scope.current_page = $scope.form_pages[$scope.currentPageIdx];
+        } else if ($scope.currentPageIdx == 0) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/grischa/mytardis/file/501734623/issues/source?bid=2125231&fileBranchId=2256393#l377)